polaris/benchmark/_base.py

```diff
@@ -182,30 +182,44 @@ def _validate_split(cls, v, info: ValidationInfo):
             raise InvalidBenchmarkError("The predefined split contains empty test partitions")
 
         train_idx_list = v[0]
-        test_idx_list = list(i for part in v[1].values() for i in part) if isinstance(v[1], dict) else v[1]
+        full_test_idx_list = (
+            list(i for part in v[1].values() for i in part) if isinstance(v[1], dict) else v[1]
+        )
```

Comment:
While not directly related to the main focus of this PR, I noticed we could potentially optimize this line by using [`itertools.chain.from_iterable`](https://docs.python.org/3/library/itertools.html#itertools.chain.from_iterable):

```suggestion
        full_test_idx_list = list(chain.from_iterable(v[1].values())) if isinstance(v[1], dict) else v[1]
```